### ISSUE TITLE
TAG Issue, critical tag missing - PAD Assignment

### ISSUE THREAD

**Issue by sbuchanYVR** [Helipaddy/report#3]
We Need ‘**PAD assignment by Radio on Approach**’.  Most Heliports require a UNICOM radio call-on approach for PAD assignments.   Note that the PPR is often obtained hours or a day ahead.  The PPR and the PAD assignment are different things.

**Comment by paddywills** [comment]
I think this is for airfields/heliports mainly?

**Comment by sbuchanYVR** [comment]
No.  Sites like April Point and Sanora (Hotels) will often guide to parking on Radio .. this avoids an engine start. On Oct 5, 2023, at 1:18 AM, Paddy Wills ***@***.***> wrote:﻿
I think this is for airfields/heliports mainly?
